scenarios/tracking/02_evaluating_mot_challenge.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<p>First sentence has typo: This notebook is used to evaluate or This notebook is for evaluating</p><p> </p><p> </p><p>"These datasets contain many video sequences, with different tracking difficulty levels, with annotated ground-truth." -&gt; These datasets contain various video annotated sequences, each with different tracking difficulty levels.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>